split-gpg2-client

```diff
@@ -1,21 +1,35 @@
-#!/bin/bash
+#!/bin/bash --
+set -euo pipefail
 
 for d in /etc "${XDG_CONFIG_HOME:-$HOME/.config}"; do
     rc_file="$d/split-gpg2-rc"
-    if [ -r "$rc_file" ]; then
+    if [[ -r "$rc_file" ]]; then
         . "$rc_file"
     fi
 done
 
-if [[ -z "$SPLIT_GPG2_SERVER_DOMAIN" ]]; then
+if ! [[ -v SPLIT_GPG2_SERVER_DOMAIN ]]; then
```

Comment:
This would break "unsetting" with SPLIT_GPG2_SERVER_DOMAIN="", which is unusual for shell scripts.

Reply:
Agreed, omitted.